client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -186,8 +186,8 @@ class JetpackChecklist extends PureComponent< Props & LocalizeProps > {
 					{ isPaidPlan && isRewindUnavailable && productInstallStatus && (
 						<Task
 							id="jetpack_vaultpress"
-							title={ translate( "We're automatically turning on Backup and Scan." ) }
-							completedTitle={ translate( "We've automatically turned on Backup and Scan." ) }
+							title={ translate( "We're automatically turning on VaultPress." ) }
+							completedTitle={ translate( "We've automatically turned on VaultPress." ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'We've automatically turned on spam filtering.' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/change-vp-checklist-wording)